src/@next/AuthenticationInput/AuthenticationInputSingle.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useRef } from 'react';
+import { NumberInput } from '../NumberInput';
+
+interface AuthenticationInputSingleProps
+  extends Omit<React.HTMLAttributes<HTMLInputElement>, 'onChange'> {
+  index: number;
+  value: number | '';
+  shouldFocus?: boolean;
+  error?: boolean;
+  disabled?: boolean;
```

Comment:
can we change the cursor pointer when its disabled? for better semantics

Reply:
done